aea/crypto/base.py

```diff
@@ -67,7 +68,11 @@ def public_key(self) -> str:
 
         :return: a public key string in base58 format
         """
-        return self._public_key_b58
+        fetch_crypt = FetchCrypto()
```

Comment:
`fetch_crypt` > `fetch_crypto`

Reply:
However, I would initialize this and the entire public keys dictionary in the `__init__` method and just call the initialized dict here